src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts

```diff
@@ -62,13 +61,16 @@ registerAction2(class extends Action2 {
 		if (walkthroughID) {
 			const selectedCategory = typeof walkthroughID === 'string' ? walkthroughID : walkthroughID.category;
 			const selectedStep = typeof walkthroughID === 'string' ? undefined : walkthroughID.step;
+			const editor = instantiationService.createInstance(GettingStartedInput,
```

Comment:
This seems not needed and should go back to how it was now that the resolver is contributed on startup.